### DISCUSSION TITLE
Allow importing plugins individually

### DISCUSSION THREAD

**Discussion by ryanb** [orchidjs/tom-select#222] [Ideas]
I can't find a good way to import a custom list of plugins using Webpack. I saw #68 but I don't think the solution presented there is ideal since it includes all dependencies for each plugin.

It appears the main issue is each plugin imports a relative `../../tom_select` file but that can't be found in `dist`.

I suggest removing the `TomSelect.define` clause from each plugin and setting it in the bundled js. For example:

```js
import TomSelect from './tom-select';
TomSelect.define("caret_position", require('./plugins/caret_position/plugin'));
TomSelect.define("dropdown_input", require('./plugins/dropdown_input/plugin'));
...
```

This removes the relative `../../tom_select` dependency so we can import them in a similar manner.

**Comment by oyejorge** [comment]
Any interest in putting a PR together?

**Reply by ryanb** [reply]
@oyejorge done here: #223

BTW, thanks for your work on tom-select! I'm transitioning from select2 and glad to have found a non-jQuery alternative.
